### PR TITLE
docs making clear there are 2 different rsync

### PR DESCRIPTION
**Pull request by bilogic**
as discussed on #2568
Is there a way to link from `api.md` to `rsync.md`?
I would like to link `upload()` and `download()`.

### CODE REVIEW

File: docs/rsync.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# rsync
+
+[Source](/src/Utility/Rsync.php)
+
+## IMPORTANT
+
+This is the deployer's built-in rsync and must not be confused with `/contrib/rsync.php`. Their configuration options are also very different, read carefully below.
+
+## Configuration options
```
Comment (antonmedv) [inline docs/rsync.md:9]: Better to put it next to upload/download func.

Comment (bilogic) [reply docs/rsync.md:9]: ok, what is the syntax to link from `api.md` to `rsync.md`?

Comment (bilogic) [reply docs/rsync.md:9]: @antonmedv any idea? thanks

Comment (Schrank) [reply docs/rsync.md:9]: [TEXT](LINK TARGET)
e.g.[Source](/src/Utility/Rsync.php)

Does this help?

Comment (antonmedv) [reply docs/rsync.md:9]: Also I don’t like wording. It’s not an external docs. “Their” not fit here.

Comment (bilogic) [reply docs/rsync.md:9]: @Schrank I'm not sure, I could not find any references around, but let me try
@antonmedv Let me find some time to fix it all up

**Pull request merged**